tests/bug-detectors/eval/tests.fuzz.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("eval", () => {
+	it.fuzz("No error", (data) => {
+		eval(
```

Comment:
Import test functions from `fuzz.js` and use them as test bodies here? IMHO the current state is clearer, but the code duplication could lead to problems down the line.